### DISCUSSION TITLE
How to add Tailwind 3 in Vue 3?

### DISCUSSION THREAD

**Discussion by jamesli2021** [tailwindlabs/tailwindcss#9544] [Help]
I get eresolve errror when doing `vue create app1` and `npm install -D tailwindcss@latest postcss@latest autoprefixer@latest`

I assume `vue create` command provide the option to add vue-router which I need, or Tailwind need to resolve dependencies?

How do you get it to work or adding `--force` should be fine?

```
npm install postcss@latest
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: @vue/eslint-config-standard@6.1.0
npm ERR! Found: eslint-plugin-vue@8.7.1
npm ERR! node_modules/eslint-plugin-vue
npm ERR!   dev eslint-plugin-vue@"^8.0.3" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer eslint-plugin-vue@"^7.0.0" from @vue/eslint-config-standard@6.1.0
npm ERR! node_modules/@vue/eslint-config-standard
npm ERR!   dev @vue/eslint-config-standard@"^6.1.0" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: eslint-plugin-vue@7.20.0
npm ERR! node_modules/eslint-plugin-vue
npm ERR!   peer eslint-plugin-vue@"^7.0.0" from @vue/eslint-config-standard@6.1.0
npm ERR!   node_modules/@vue/eslint-config-standard
npm ERR!     dev @vue/eslint-config-standard@"^6.1.0" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /Users/s/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/s/.npm/_logs/2022-10-13T17_10_24_519Z-debug-0.log
```

**Comment by hardikdangar** [comment]
What node version you are using ? i am assuming there is issue with old version of nodejs you are using with current dependencies. you can try 
npm install --save --legacy-peer-deps
but it could be broken installation.

**Reply by jamesli2021** [reply]
Node version, recently installed.
16.17.1

`--legacy-peer-deps` works for me when I tested it but it does seem like a proper approach for use for a new project.

I have a search on Reddit and read that `vue create` is now considered a maintainence mode.
Has been superseded by vite using `create-vue@latest` as the modern tool? 

```
npx create-vue@latest

cd app2
npm install -D tailwindcss postcss@latest autoprefixer
npx tailwindcss init -p
```

**Reply by hardikdangar** [reply]
official docs suggest this way,

https://vuejs.org/guide/quick-start.html#creating-a-vue-application

**Reply by jamesli2021** [reply]
Appreciate, I did try that docs too, the questions look similar and it uses vite as well.

Found the actual guide
https://github.com/vitejs/vite/tree/main/packages/create-vite#readme
